pkg/daemon/daemon.go

```diff
@@ -53,6 +53,7 @@ type States struct {
 	// Where the configuration file resides, all rafs instances share the same configuration template
 	ConfigDir      string
 	SupervisorPath string
+	PrefetchFiles  string
```

Comment:
Is this resolved? I suppose it's not. You can put the prefetch info into DB definitely. I am a little tired of such things

Reply:
I also think that saving the `PrefetchFiles` to DB is not a good choice. You can move it to `Rafs` struct if this value is different to each instance. If we have to store it in the DB, please provide more details. TKS.

Reply:
Hello, thanks for your review. According to my understanding, adding PrefetchFiles to States is to persist this parameter. If PrefetchFiles is not added to States, when snapshotter unexpectedly restarts and triggers the recovery process after running the nydusd command, the BuildDaemonCommand method will discard the --prefetch-files parameter when rebuilding the nydusd command. （PS: I am sooo sorry that many of my previous replies to comments were in a pending state, so that you cannot see them. I will reply again, I'm really really sorry. ）

Reply:
I am sooo sorry that many of my previous replies to comments were in a pending state, so that you cannot see them. I will reply again, I'm really really sorry.